libs.freemarker/external/freemarker-2.3.19-notice.txt

```diff
@@ -1,6 +1,35 @@
-Apache FreeMarker (incubating)
-Copyright 2015-2017 The Apache Software Foundation
+Copyright (c) 2003 The Visigoth Software Society. All rights reserved.
 
-This product includes software developed at
-The Apache Software Foundation (http://www.apache.org/).
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
```

Comment:
I would not go for the full license in the notice. I would just add the required part (see section 2 of the license):

> This product includes software developed by the
> Visigoth Software Society (http://www.visigoths.org/).

For updates please also squash the changes together - there will now be at least four changeset for more or less one change.